src/recipe/Baked.Recipe.Service/Communication/Response.cs

```diff
@@ -3,10 +3,16 @@
 namespace Baked.Communication;
 
 public record Response(
+    StatusCode StatusCode,
     string Content
 )
 {
+    public static Response Success(string content) =>
+        new(StatusCode.Success, content);
```

Comment:
if you created this one be consistent and create `Handled()`, and `Unhandled()` as well, or remove this one